include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -183,6 +183,20 @@ struct __init_processing
 //------------------------------------------------------------------------
 // transform_reduce
 //------------------------------------------------------------------------
+template <typename... _Ts>
+std::tuple<_Ts...>
+select_from_group(sycl::sub_group G, std::tuple<_Ts...> value, unsigned int idx)
+{
+    return std::apply([G, idx](auto&&... elems) { return std::make_tuple(sycl::select_from_group(G, elems, idx)...); },
+                      value);
+}
+
+template <typename _T>
+_T
+select_from_group(sycl::sub_group G, _T value, unsigned int idx)
+{
+    return sycl::select_from_group(G, value, idx);
```

Comment:
It may be good to change this to use the `__` and `_` formatting

Reply:
Good suggestion, done.